src/pseudopeople/loader.py

```diff
@@ -7,21 +7,40 @@
 
 from pseudopeople.constants.metadata import DatasetNames
 from pseudopeople.exceptions import DataSourceError
+from pseudopeople.utilities import PANDAS, DataFrame, Engine
 
 
-def load_standard_dataset_file(data_path: Path, user_filters: List[Tuple]) -> pd.DataFrame:
-    if data_path.suffix == ".parquet":
+def load_standard_dataset(
+    data_path: Path,
+    user_filters: List[Tuple],
+    engine: Engine = PANDAS,
+    is_file: bool = True,
+) -> DataFrame:
+    if is_file and data_path.suffix != ".parquet":
```

Comment:
Why are we not doing this check if `not is_file`?

Reply:
Because when we load an entire Parquet directory using Modin, the data path here would be e.g. `/path/to/simulated/pop/data/decennial_census/`. It actually is a standard practice to give Parquet directories names that end in `.parquet` but we haven't done that, so there is no check to do here.